src/geom-functions.hpp

```diff
@@ -170,12 +170,12 @@ geometry_t centroid(geometry_t const &geom);
  *
  * Returns null geometry for anything but linestring geometries!
  *
- * \param output Pointer to output geometry.
+ * \param input Pointer to output geometry.
  * \param geom Input geometry.
```

Comment:
You changed the wrong parameter name here.

Reply:
damn... fixed in new version of this PR